resources/linux/code.appdata.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Checked, and fixed a couple of issues. Right now it says:

```
vscode % appstream-util validate resources/linux/code.appdata.xml
resources/linux/code.appdata.xml: FAILED:
• tag-missing           : <translation> not specified
• tag-missing           : <update_contact> is not present
Validation of files failed
```

These are optional tags, and `validate-relax` succeds:

```
vscode % appstream-util validate-relax resources/linux/code.appdata.xml
resources/linux/code.appdata.xml: OK
vscode % 
```